.eleventy.js

```diff
@@ -1,34 +1,41 @@
-const markdownIt = require('markdown-it');
-const emoji = require('markdown-it-emoji');
+const markdownIt = require("markdown-it");
+const emoji = require("markdown-it-emoji");
 
-const {isAfter, isBefore} = require('date-fns');
+const { isAfter, isBefore, isToday } = require("date-fns");
 
 module.exports = (eleventyConfiguration) => {
-	eleventyConfiguration.addCollection('events', (collectionApi) => {
-		return collectionApi.getFilteredByGlob('./src/schedule/*.md');
-	});
-	eleventyConfiguration.addCollection('pastEvents', (collectionApi) => {
-		return collectionApi
-			.getFilteredByGlob('./src/schedule/*.md')
-			.filter(event => isBefore(new Date(event.data.date), new Date()));
-	});
-	eleventyConfiguration.addCollection('upcomingEvents', (collectionApi) => {
-		return collectionApi
-			.getFilteredByGlob('./src/schedule/*.md')
-			.filter(event => isAfter(new Date(event.data.date), new Date()));
-	});
+  eleventyConfiguration.addCollection("events", (collectionApi) => {
+    return collectionApi.getFilteredByGlob("./src/schedule/*.md");
+  });
+  eleventyConfiguration.addCollection("pastEvents", (collectionApi) => {
+    return collectionApi
+      .getFilteredByGlob("./src/schedule/*.md")
+      .filter((event) => isBefore(new Date(event.data.date), new Date()));
+  });
+  eleventyConfiguration.addCollection("upcomingEvents", (collectionApi) => {
+    // We include `isToday` assuming that a daily build job is ran at 12:01am,
+    // and we're okay with a daily event potentially showing even if it's in the past
+    // _(until 12:00:59am of the next day)_
+    return collectionApi
+      .getFilteredByGlob("./src/schedule/*.md")
+      .filter(
+        (event) =>
+          isAfter(new Date(event.data.date), new Date()) ||
+          isToday(new Date(event.data.date))
+      );
+  });
```

Comment:
Comment mostly explains the functionality. By default, anything in the future evaluates to true. If something is in the past, but still today (in the server's TZ), it'll show up. I don't think we'd need to deal with TZ's on the server here, but if necessary, we can do that as well.

Reply:
This comment is great, and I think it'll help people who come to this codebase later.